rxandroidble/src/main/java/com/polidea/rxandroidble2/scan/ScanResultInterface.java

```diff
@@ -0,0 +1,13 @@
+package com.polidea.rxandroidble2.scan;
+
+public interface ScanResultInterface {
```

Comment:
How can this be done? (Has to be public as public classes implement it, is there an annotation I should use?)

Reply:
Since method/interface visibility is not a trivial problem the usual approach is to put interfaces that are not meant for external usage under `internal` package. It can still be public but this should hint the user to not use it :)